src/vtbackend/Terminal.cpp

```diff
@@ -263,12 +263,12 @@ bool Terminal::processInputOnce()
 
     auto const readResult = readFromPty();
 
-    if (!readResult)
+    if (!readResult.has_value())
```

Comment:
this should be equivalent. I am surprised and wondering why you did that. :)

Reply:
i was thinking that pointer to bool conversion been happening, was it  internal conversion to bool from std::optional  side ?

Reply:
Oh, I see, indeed it has operator bool()

Reply:
Updated